src/index.html

```diff
@@ -1,12 +1,354 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#" class="header__logo-link">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <div class="nav__link">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </div>
+          </li>
+          <li  class="nav__list-item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__list-item">
+            <a
+            href="#"
+            class="nav__link"
+            data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page__main">
+      <div class="container">
+        <div class="card" data-qa="card">
+          <img
+          src="./images/imac.jpeg"
+          class="card__image"
+          alt="Apple A1419"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <div class="card__product-code">
+            Product code: 195434
+          </div>
```

Comment:
Do not use div for text content - use semantic tags instead, `p` for example. Fix all similar cases